Sources/FluidAudio/DownloadUtils.swift

```diff
@@ -98,7 +98,7 @@ public class DownloadUtils {
 
     /// Model repositories on HuggingFace
     public enum Repo: String, CaseIterable {
-        case vad = "FluidInference/silero-vad-coreml"
+        case vad = "FluidInference/silero-vad-v2-coreml"
```

Comment:
Can we use the same HF repo and just upload it to the folder there differently.

Reply:
If the previous version doesn't really work I don't see why we need to keep it around